README.md

```diff
@@ -1,7 +1,8 @@
-# TiDB Community
```

Comment:
Why we remove this title?

/hold

Reply:
Duplicated with "TiDB Community Logo", the style is copied from: https://github.com/pingcap/tidb#readme

Reply:
Preview: https://github.com/pingcap/community/tree/sykp241095-patch-4#readme